eZ/Publish/API/Repository/Tests/FieldType/ImageIntegrationTest.php

```diff
@@ -760,6 +763,108 @@ public function testRemovingDraftRemovesOldImage(): void
         );
     }
 
+    public function testDeleteImageWithCorruptedName(): void
+    {
+        $content = $this->publishNewImage(
+            __METHOD__,
+            new ImageValue(
+                [
+                    'inputUri' => __DIR__ . '/_fixtures/image.jpg',
+                    'fileName' => 'image.jpg',
+                    'fileSize' => filesize(__DIR__ . '/_fixtures/image.jpg'),
+                    'alternativeText' => 'Alternative',
+                ]
+            ),
+            [2]
+        );
+
+        $imageFieldDefinition = $content->getContentType()->getFieldDefinition('image');
+
+        $record = $this->fetchXML(
+            $content->id,
+            $content->getVersionInfo()->versionNo,
+            $imageFieldDefinition->id
+        );
+
+        $document = $this->corruptImageFieldXML($record);
+
+        $this->updateXML(
+            $content->id,
+            $content->getVersionInfo()->versionNo,
+            $imageFieldDefinition->id,
+            $document
+        );
+
+        $repository = $this->getRepository(false);
+        $contentService = $repository->getContentService();
+
+        $contentService->deleteContent($content->getVersionInfo()->getContentInfo());
+
+        // Expect no League\Flysystem\CorruptedPathDetected thrown
```

Comment:
Assert no assertions are made here (unless I'm mistaken?).

Reply:
> Assert no assertions are made here (unless I'm mistaken?).

Either there are some implicit assertions performed in the background or we don't have strict config. We'll see how it behaves on a merge up.